src/ai-bundle/src/Profiler/DataCollector.php

```diff
@@ -38,16 +41,32 @@ final class DataCollector extends AbstractDataCollector implements LateDataColle
     private readonly array $toolboxes;
 
     /**
-     * @param TraceablePlatform[] $platforms
-     * @param TraceableToolbox[]  $toolboxes
+     * @var MessageStoreInterface[]
+     */
+    private readonly array $messageStores;
+
+    /**
+     * @var ChatInterface[]
+     */
+    private readonly array $chats;
+
+    /**
+     * @param TraceablePlatform[]     $platforms
+     * @param TraceableToolbox[]      $toolboxes
+     * @param TraceableMessageStore[] $messageStores
+     * @param TraceableChat[]         $chats
```

Comment:
```suggestion
     * @param TraceablePlatform[] $platforms
     * @param TraceableToolbox[] $toolboxes
     * @param TraceableMessageStore[] $messageStores
     * @param TraceableChat[] $chats
```

I think we should add a php-cs-fixer rule for this, no?

Reply:
Let's stick to Symfony standard here